src/db/tables/breaches.js

```diff
@@ -18,6 +18,20 @@ async function getAllBreaches() {
     .returning("*")
 }
 
+/**
+ * Get all count from "breaches" table
+ *
+ * @returns Count of all records from "breaches" table
+ */
+async function getAllBreachesCount() {
+  const breachesCount = await knex('breaches')
+    .count({ count: "*" })
```

Comment:
This table is not very large, but if it became a problem we could consider doing an index-only scan (e.g. if the index is likely to be significantly smaller than the table).

Reply:
Good suggestion! Made the change in [bd3a6df](https://github.com/mozilla/blurts-server/pull/3233/commits/bd3a6df8c2ed1e6c2e59780192ad8b0f2fc822d6).